### ISSUE TITLE
response multiple array

### ISSUE THREAD

**Issue by webgori** [OAI/OpenAPI-Specification#739]
my swagger.yaml is here

```
  responses:
    200:
      description: An array of Students
      schema:
        type: array
        items:
          $ref: '#/definitions/Students'
definitions:
  Students:
    type: array
    items:
      $ref: '#/definitions/Student'
  Student:
    properties:
      name:
        type: string
```

but, result is
![image](https://cloud.githubusercontent.com/assets/5680005/16937698/37371e78-4dac-11e6-8dc9-67a4ec9728c2.png)
i want to change like this

```
{
  "Students":
  [
     [1, 2, "tom", true]
  ],
  "OtherOne":[true, 1, "test"]
}
```

thank you for read @__@

**Comment by baileylo** [comment]
I'm not exactly sure what you want vs what you are currently seeing. But this should get you the output you have specified.

```yaml
schema:
  type: object
  properties:
    Students: { $ref: '#/definitions/Students }
   OtherOne: { $ref: '#/definitions/Studen}
```

**Comment by webgori** [comment]
@baileylo need more detail /definitions/Students and /definitions/Studen for `[1, 2, "tom", true]` and `[true, 1, "test"]`

**Comment by handrews** [comment]
This seems to have fizzled out more than three years ago, so I'm going to assume no one's waiting on it.  Please re-open or open a new issue if needed.
